metrics/ci_integrations/test/source/github_actions/adapter/github_actions_source_client_adapter_test.dart

```diff
@@ -273,6 +274,25 @@ void main() {
       },
     );
 
+    test(
+      ".fetchBuilds() does not fetch builds of the queued workflow runs",
```

Comment:
```suggestion
      ".fetchBuilds() returns an empty list if the client returns workflow runs with the queued status",
```
Does it make sense? Also, I think this suggestion is applicable for the `.fetchBuilds() does not fetch builds of the queued jobs` and other similar tests.